ReactAndroid/src/main/java/com/facebook/react/ReactInstanceManager.java

```diff
@@ -88,6 +88,11 @@
   public abstract void onBackPressed();
 
   /**
+   * This method will give JS the opportunity to receive intents via Linking.
+   */
+  public abstract void onNewIntent(Intent intent);
```

Comment:
@andreicoman11 It doesn't exist in the `ActivityEventListener` interface, so can't be used in a module. Should we add it there?